static/usage/v7/datetime/styling/calendar-days/angular/example_component_html.md

```diff
@@ -0,0 +1,3 @@
+```html
```

Comment:
Let's turn off the emulated view encapsulation the as Sean's [comment](https://github.com/ionic-team/ionic-docs/pull/3101#issuecomment-1696566830) mentions

Reply:
Updated both calendar days & wheel styling examples to turn off ViewEncapsulation: https://github.com/ionic-team/ionic-docs/pull/3101/commits/fa730ff56d29d0426ea61bf5d419b3ff10aacf97